src/containers/Question.jsx

```diff
@@ -1,15 +1,18 @@
-import React, { Component } from 'react';
+import React, { Component, PropTypes } from 'react';
 import C from '../constants';
 import Firebase from 'firebase';
 import store from '../store';
 import { connect } from 'react-redux';
-
+import isPlainObject from 'lodash/lang/isPlainObject';
```

Comment:
do u really need to import specified isPlanObject function like this ?
is there any special reason.

In normal case we just need to import lodash and call `lodash.isPlainObject`

```